kong/clustering/data_plane.lua

```diff
@@ -18,6 +18,7 @@ local tostring = tostring
 local ngx = ngx
 local ngx_log = ngx.log
 local ngx_sleep = ngx.sleep
+local ngx_process = ngx.process
```

Comment:
should be `local process_type = ngx.process.type`

Reply:
done